docker/monitoring/prometheus-micrometer.yaml

```diff
@@ -0,0 +1,6 @@
+scrape_configs:
```

Comment:
There is an already existing `prometheus.yaml` file next to this. Since that one is mounted and this one is not, the job description should be added to that file.